src/main/io/vtx_smartaudio.c

```diff
@@ -70,6 +71,25 @@ static void saUpdateStatusString(void); // Forward
 
 static serialPort_t *smartAudioSerialPort = NULL;
 
+#if defined(CMS) || defined(VTX_COMMON)
+static const char * const saPowerNames[] = {
+    "---", "25 ", "200", "500", "800",
+};
+#endif
+
+#ifdef VTX_COMMON
+static vtxVTable_t saVTable;    // Forward
```

Comment:
Learned today that this is legal in C ...

This is declaration and *tentative* definition - if actual definition in this file gets removed, `saVTable` will be defined here, containing all zeroes. No warning will be produced.
Combined with `#ifdef`s, this is pretty dangerous construction ... 

IMO it is better to pollute global namespace - declare it `extern` here and remove `static` in actual definition. Linker will catch missing definition then ... 
(or reorder definitions ... )